cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -287,23 +287,23 @@ The following commands are only executable against the `system` database:
 | <<administration-security-users-drop, DROP USER ... [IF EXISTS]>>                                                                                                                 | User and role administration  | Deletes a specified user.
-| <<administration-security-users-alter, ALTER USER ... [SET PASSWORD ...] [SET PASSWORD CHANGE [NOT] REQUIRED] [SET STATUS [ACTIVE \| SUSPENDED]]>>                                | User and role administration  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
+| <<administration-security-users-alter, ALTER USER ... [SET PASSWORD {password [CHANGE [NOT] REQUIRED] | CHANGE [NOT] REQUIRED}] [SET STATUS {ACTIVE | SUSPENDED}]>>               | User and role administration  | Changes a user account. Changes can include setting a new password, setting the account status and enabling that the user should change the password upon next login.
 | <<administration-security-users-alter-password, ALTER CURRENT USER SET PASSWORD FROM ... TO>>                                                                                     | User and role administration  | Change the password of the user that is currently logged in.
-| <<administration-security-subgraph-show, SHOW [ROLE ... \| USER ... \| ALL] PRIVILEGES>>                                                                                          | User and role administration  | Returns information about role, user or all privileges.
-| <<administration-security-roles-grant, GRANT ROLE[S] ... TO>>                                                                                                                     | Privilege administration      | Assigns one or multiple roles to one or multiple users.
-| <<administration-security-administration-database-privileges, GRANT ... ON DATABASE ... TO>>                                                                                      | Privilege administration      | Assigns a database, schema, or token privilege to one or multiple roles.
+| <<administration-security-roles-grant, GRANT ROLE[S] ... TO>>                                                                                                                     | User and role administration  | Assigns one or multiple roles to one or multiple users.
+| <<administration-security-roles-revoke, REVOKE ROLE[S] ... FROM>>                                                                                                                 | User and role administration  | Removes one or multiple roles from one or multiple users.
+| <<administration-security-subgraph-show, SHOW {ROLE ... \| USER ... \| ALL } PRIVILEGES>>                                                                                         | Privilege administration      | Returns information about role, user or all privileges.
```

Comment:
Now you made it not optional... it was optional before and I missed it :P (now its a group where you should choose one)